samples/csharp_dotnetcore/05.multi-turn-prompt/AdapterWithLogging.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public class AdapterWithLogging : BotFrameworkHttpAdapter
```

Comment:
Maybe something like AdapterWithErrorHandling

Reply:
OK makes sense. We just copied this class from the other project. I'l rename them together.